cmd/ipfs/daemon.go

```diff
@@ -36,6 +37,7 @@ the daemon.
 	Options: []cmds.Option{
 		cmds.BoolOption(initOptionKwd, "Initialize IPFS with default settings if not already initialized"),
 		cmds.BoolOption(mountKwd, "Mounts IPFS to the filesystem"),
+		cmds.BoolOption(readOnlyKwd, "Make the HTTP gateway read only (disable POST, PUT and DELETE)"),
```

Comment:
let's flip this-- i think the default should be read only and writing should need enabling (at least for the time being). maybe:

```
-w --writable "Enable writing objects (with POST, PUT, and DELETE)"
```